README.md

```diff
@@ -1 +1,17 @@
-# codacy-test
\ No newline at end of file
+## Codacy ESLint
+
+### Among Codacy’s features
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://app.codacy.com/gh/franciscodua/codacy-test/pullRequest?prid=7573067)
```suggestion

- Identify new Static Analysis issues
```